src/globalContext.ts

```diff
@@ -64,7 +72,8 @@ export async function getGlobalContext(
     } = windowObject[0] as any;
 
     // Storybook 6.4 doesn't have the store property. Use the storyStore instead.
-    const foundStore = store?.() || storyStore;
+    // Storybook 8 doesn't have storyStore property. Use preview's storyStoreValue instead.
+    const foundStore = store?.() || storyStore || storyStoreValue;
```

Comment:
If we're not adding this [v9 breaking change](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#storystore-and-methods-deprecated) now, it's better to at least comment about it.